src/docs/containers/color-palette.less

```diff
@@ -88,32 +89,30 @@
 	color-chart-bad,
 	color-chart-bad-dark,
 	color-chart-bad-light,
-	color-black;
+	color-chart-neutral;
 
 .ColorPalette-loop(@i: length(@ColorPalette-variables)) when (@i > 0) {
 	@name: extract(@ColorPalette-variables, @i);
 
-	&-@{name} { background-color: @@name }
+	&-@{name} {
+		background-color: @@name;
+		color: contrast(@@name);
+	}
 
 	// Recurse
 	.ColorPalette-loop(@i - 1);
 }
 
-.lucid-ColorPalette {
+.ColorPalette {
+	padding: 15px;
```

Comment:
Yeah probably, I use it in lots of places. I've been cutting some corners on the demo page :/